### DISCUSSION TITLE
v0.20.6

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1535] [Releases]
> [!WARNING]  
> `v0.20.x` has breaking changes and requires action. Read the original release notes here: https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.20.0

## What's Changed
* Update README.md by @svenvg93 in https://github.com/alexjustesen/speedtest-tracker/pull/1525
* Fix issue with Notify on every speedtest run failing  by @tgregory86 in https://github.com/alexjustesen/speedtest-tracker/pull/1521
* Add Download & Upload Latency graphs to the public dashboard by @svenvg93 in https://github.com/alexjustesen/speedtest-tracker/pull/1531
* [Chore] Swapped download and upload jitter charts by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1532
* [Feature] Write the server id to the result on failed tests by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1533
* Release v0.20.6 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1534

## New Contributors
* @tgregory86 made their first contribution in https://github.com/alexjustesen/speedtest-tracker/pull/1521

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.20.5...v0.20.6

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.20.6'>v0.20.6</a>.</em>

**Comment by Skyeci73** [comment]
anyone else seeing issues with slower than expected upload tests results since the last couple of updates? My 2b/2gb has dropped to 1.6gb on the upload test. I have checked it against different server id's via the stt which give a lower result than the ookla app which gives full speed result regardless of which server I test against.

Is it possible to roll back to an earlier version?

**Reply by svenvg93** [reply]
Nothing has changed in the speedtest client itself in the last releases.
If you want you can change the version of the container to a older one to give it a try of course.
`lscr.io/linuxserver/speedtest-tracker:0.19.0`

Keep the changes of 0.20.x in mind when downgrading and upgrading agian 😉 
Maybe it would be easier to create a new container with a older version next the current one to compare.

**Reply by Skyeci73** [reply]
Thanks that's a good idea. I will try that tonight and post back if it's any different.

**Reply by svenvg93** [reply]
Just make sure both containers don't test at the same time 😉

**Reply by Skyeci73** [reply]
Hi. Can I check the old version  is still active. I'm still looking at this and need to test it now with the older version. Basically with the newer version and a 10gb rj45 card the upload peaks at 1.25gb. Soon as I put the 2.5gb lan card back in the upload goes back to 1.9/2.0 on the same machine. My lan is 10gb cat6.

Thanks

**Comment by skyeci-draytek** [comment]
well I finally got to bottom of it. My isp connection is 2GB/2GB. My lan has a 10gb switch and some of my devices including my speedtest ubuntu server have 10gb nics. Anyway after a lot of head scratching I finally cracked it. The main issue was recently I was finding the upload speed test results were hitting about 1.5gb regardless of when I tried to run a test. My router is a pfsense box with a 2.5gb nic for the isp connection and a 10gb card for the lan side. In the end I matched network card types for the routers 10gb lan side to the speed test machine. Since doing that I am now getting faster latency results and the upload/download are hitting maxium speed. Seeme like with the cards being different manufactures they were not getting on..both now on 10gtek 10gb rj45 nics and no further issues. yay! @svenvg93 @alexjustesen 

![spt1](https://github.com/user-attachments/assets/31250237-0fb8-4437-8870-8163838a0f04)
